patch_rebaser/patch_rebaser.py

```diff
@@ -35,6 +35,40 @@ def find_patches_branch(repo, remote, distgit_branch):
     return None
 
 
+def create_patches_branch(repo, commit, remote, dev_mode=True):
+    """Create new patches branch from commit"""
+    branch_name = os.environ.get('PATCHES_BRANCH', None)
```

Comment:
Here I'm taking advantage of the PATCHES_BRANCH env var created by the patch_rebaser.sh script. I think a better idea would be to add a new parameter to the ini file with the patches branch.

That would create some side-effects, like not having to guess the -patches branch in find_patches_branch(), but that should be handled by a separate commit.

WDYT?

Reply:
Ack. We are assuming many other env vars are present (all the DLRN_* ones), so let's use it.